Sources/Danger/Plugins/SwiftLint/SwiftLintViolation.swift

```diff
@@ -20,7 +20,8 @@ public struct SwiftLintViolation: Decodable {
     }
 
     public func toMarkdown() -> String {
-        let formattedFile = file.split(separator: "/").last! + ":\(line)"
+        let filename = file.split(separator: "/").last ?? "unknown"
+        let formattedFile = filename + ":\(line)"
```

Comment:
```suggestion
        let formattedFile = "\(filename):\(line)"
```

IMO, unifying to interpolation or concatenation is better than mixed.